build/rpms/BUILD

```diff
@@ -4,6 +4,7 @@ load("@bazel_tools//tools/build_defs/pkg:rpm.bzl", "pkg_rpm")
 
 pkg_rpm(
     name = "kubectl",
+    architecture = "x86_64",
```

Comment:
Bazel supports multiple platforms?

Reply:
kinda sorta soon

I don't think it's 100% working yet.

Reply:
that argument in case of pkg_rpm just used for filename generation.